code/modules/virus2/disease2.dm

```diff
@@ -16,11 +17,17 @@ var/global/list/disease2_list = list()
 	var/logged_virusfood=0
 
 /datum/disease2/disease/New(var/notes="No notes.")
-	uniqueID = rand(0,10000)
+	generate_uniqueID()
 	log += "<br />[timestamp()] CREATED - [notes]<br>"
-	disease2_list["[uniqueID]"] = src
 	..()
 
+/datum/disease2/disease/proc/generate_uniqueID()
+	for (var/x = 1 to 5)
+    	uniqueID = rand(...)
```

Comment:
...Did you really copypaste the code this literally?

How the shit does `rand(...)` even compile?

When I wrote that snippet for you I assumed you'd fill in the rand...